### PR TITLE
fix: fix tx env epoch value

### PR DESCRIPTION
**Pull request by driftluo**
## What this PR does / why we need it?

This PR tx env epoch value

### What is the impact of this PR?

No Breaking Change

**PR relation**:
- Ref #

<details><summary>CI Settings</summary><br/>

### **CI Usage**

**Tip**: Check the CI you want to run below, and then comment `/run-ci`.

**CI Switch**

- [ ] Web3 Compatible Tests
- [ ] OCT 1-5 And 12-15
- [ ] OCT 6-10
- [ ] OCT 11
- [ ] OCT 16-19
- [ ] v3 Core Tests

### **CI Description**

| CI Name                                   | Description                                                               |
| ----------------------------------------- | ------------------------------------------------------------------------- |
| *Web3 Compatible Test*                    | Test the Web3 compatibility of Axon                                       |
| *v3 Core Test*                            | Run the compatibility tests provided by Uniswap V3                        |
| *OCT 1-5 \| 6-10 \| 11 \| 12-15 \| 16-19* | Run the compatibility tests provided by OpenZeppelin                      |

</details>

### CODE REVIEW

**Review by KaoImin — APPROVED** [review]

File: core/interoperation/src/lib.rs [hunk 1]
```diff
@@ -22,7 +22,9 @@ const GAS_TO_CYCLE_COEF: u64 = 6_000;
 // The following information is from CKB block [10976708](https://explorer.nervos.org/block/10976708)
 // which is CKB2023 disabled.
 const CKB2023_DISABLED_NUMBER: u64 = 10_976_708;
-const CKB2023_DISABLED_EPOCH: u64 = 0x53c007f0020c8;
+// The value must be less than the epoch of the 2023 hardfork and greater than
+// the epoch of the 2021 hardfork
+const CKB2023_DISABLED_EPOCH: u64 = 0x20c8;
```
Comment (chaoticlonghair) [inline core/interoperation/src/lib.rs:27]: Sorry, I didn’t catch your meaning.

Where does `CKB2023_DISABLED_NUMBER` and `CKB2023_DISABLED_EPOCH` come from?

### References
- [CKB hardfork information for Testnet (from latest CKB commit)](https://github.com/nervosnetwork/ckb/blob/5975fda61bd3a03ec2a4a6075a4e627f185b4d2f/util/constant/src/hardfork/testnet.rs#L4-L11)
- [CKB hardfork information for Mainnet (from latest CKB commit)](https://github.com/nervosnetwork/ckb/blob/5975fda61bd3a03ec2a4a6075a4e627f185b4d2f/util/constant/src/hardfork/mainnet.rs#L4-L11)

Comment (driftluo) [reply core/interoperation/src/lib.rs:27]: `10_976_708` this existing block must be activated in 2021 but not activated in 2023

**Review by Flouse — APPROVED** [review]

File: core/interoperation/src/lib.rs (hunk 1 above)
Comment (Flouse) [inline core/interoperation/src/lib.rs:27]: ### Question
This is the definition for CKB mainnet.
What about [CKB testnet](https://github.com/nervosnetwork/ckb/blob/4cbb826abb37a093bff9e756e4a2338efb801c81/util/constant/src/hardfork/testnet.rs#L7)?

**Comment by driftluo** [comment]
Features that will be discarded and do not need to be changed

**Pull request closed without merging**